src/Bicep.Decompiler.IntegrationTests/DecompilationTests.cs

```diff
@@ -226,5 +221,29 @@ public void Decompiler_handles_banned_function_replacement(string expression, st
 
             filesToSave[entryPointUri].Should().Contain($"output calculated {type} = ({expectedValue})");
         }
+
+        [TestMethod]
+        public void Decompiler_should_not_decompile_bicep_extension()
+        {
+            const string template = @"{
+    ""$schema"": ""https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#"",
+    ""contentVersion"": ""1.0.0.0"",
+    ""parameters"": {},
+    ""variables"": {},
+    ""resources"": [],
+    ""outputs"": {}
+}";
+
+            var fileUri = new Uri("file:///path/to/main.bicep");
+            var fileResolver = new InMemoryFileResolver(new Dictionary<Uri, string>
+            {
+                [fileUri] = template,
+            });
+
+            Action sut = () => TemplateDecompiler.DecompileFileWithModules(TestResourceTypeProvider.Create(), fileResolver, fileUri);
```

Comment:
You'll need to merge with main - I renamed this from `TestResourceTypeProvider.Create()` -> `TestTypeHelper.CreateEmptyProvider()`

Reply:
Yeah good catch. Just fixed it.